### PR TITLE
update: added responsiveness

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
Added media query to make the component library accessible for different screen sizes

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]

File: style.css [hunk 1]
```diff
@@ -20,7 +20,7 @@ body {
 
 .pagewrap {
     background-color: var(--off-white);
-    max-width: 1000px;
+    width: 90%;
     margin: auto;
```
Comment (mariyasada) [inline style.css:23]: if you want to use max-width then use vw unit

File: style.css [hunk 2]
```diff
@@ -47,9 +47,8 @@ body {
 
 .nav {
     grid-area: nav;
-    top: 0;
-    left: 0;
-    padding-top: 1rem;
+    /* top: 0;
+    left: 0; */
```
Comment (mariyasada) [inline style.css]: remove unnecessary comment

File: style.css [hunk 3]
```diff
@@ -95,24 +92,20 @@ body {
     padding: 1rem;
     display: flex;
     flex-direction: column;
-
 }
```
Comment (mariyasada) [inline style.css:92]: remove unnecessary space

File: style.css [hunk 4]
```diff
@@ -496,8 +493,8 @@ body {
...
```
Comment (mariyasada) [inline style.css:489]: remove this blank class

**Review by mariyasada — APPROVED** [review]
look at my comments.

File: style.css (hunk 1 above)
Comment (Kothalkarpriya) [reply style.css:23]: okay I will change it

File: style.css (hunk 2 above)
Comment (Kothalkarpriya) [reply style.css]: Yes, I forget about it Thanks

File: style.css (hunk 3 above)
Comment (Kothalkarpriya) [reply style.css:92]: I will do it

File: style.css (hunk 4 above)
Comment (Kothalkarpriya) [reply style.css:489]: I will do it

**Review by mariyasada — APPROVED** [review]
you can merge it

**Pull request merged**